pkg/controller/operators/catalog/operator.go

```diff
@@ -992,30 +1001,39 @@ func (o *Operator) syncConfigMap(logger *logrus.Entry, in *v1alpha1.CatalogSourc
 	}
 
 	continueSync = true
+	logger.Info("configmap catalog source sync'ed successfully")
 	return
 }
 
 func (o *Operator) syncRegistryServer(logger *logrus.Entry, in *v1alpha1.CatalogSource) (out *v1alpha1.CatalogSource, continueSync bool, syncError error) {
 	out = in.DeepCopy()
 
-	logger.Info("synchronizing registry server")
+	logger = logger.WithField("step", "syncRegistryServer")
+	logger.Info("registry server sync start")
+	defer func() {
+		logger.WithError(syncError).WithField("continueSync", continueSync).Info("registry server sync end")
```

Comment:
What happens with `WithError(syncError)` if `syncError` is nil?

Reply:
come out as <nil>

Reply:
Looking at it now, I'm wondering if we should use WithField("syncError") just to have more fidelity with the code and helping the next person that has to do a live manual debug XD